.github/workflows/unittest.yml

```diff
@@ -21,9 +21,10 @@ jobs:
       uses: coursier/cache-action@v5
 
     - name: Set up JDK 11
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v2
       with:
-        java-version: 11
+        distribution: 'adopt-hotspot'
+        java-version: '11'
```

Comment:
not necessary as part of this PR, but I was trying things and this felt good to keep in